packages/psss/src/index.js

```diff
@@ -5,12 +5,14 @@
 import React from 'react';
 import { render as renderReactApp } from 'react-dom';
 import App from './App';
+import { ReactQueryDevtools } from 'react-query-devtools';
 import { AppProviders } from './AppProviders';
 import { initStore } from './store/store';
 
 const render = () => {
   return renderReactApp(
     <AppProviders store={initStore()}>
+      <ReactQueryDevtools />
```

Comment:
Should we be turning this off in production? (honestly don't know)

Reply:
That's a good question. I hadn't thought of that and had to look it up but no it's fine to leave it like that - it turns off automatically in production.

https://github.com/tannerlinsley/react-query-devtools/blob/master/index.js